### PR TITLE
- Updated documentation

### PR DESCRIPTION
**Pull request by ThirtySomething**
Added some content and fixed warnings reported by VS Code.

### CODE REVIEW

File: Common/Readme.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+* The config loader, consisting of
+  * The generic class `ConfigLoader.cs`
+  * The interface for `IConfigObjects.cs`
+* The definition of an output plugin `IDataSink.cs`
+* The definition of an input plugin `IDataSource.cs`
+* The common used object of a `MeasurementValue.cs`
+* A generic `PluginLoader.cs`
+* An abstract base class `SensorBase.cs` for the datasource plugins
+* An abstract base class `TFSensor.cs` to access the Tinkerforge sensors
+* An utility class `TFUtils.cs` containing some helper stuff
```
Comment (DonatJR) [inline Common/Readme.md]: typo: should be "a utility class"

File: Documentation/Readme.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Documentation
```
Comment (DonatJR) [inline Documentation/Readme.md:1]: Is this file necessary if no content is included? did you create it so github shows some info when navigating to the subfolder?

**Review by DonatJR — CHANGES_REQUESTED** [review]
nice! looks good except for the one typo :+1: the other comment is just out of curiosity

File: Documentation/Readme.md (hunk 2 above)
Comment (ThirtySomething) [reply Documentation/Readme.md:1]: Basically some documentation is expected in the folder documentation. But there are only images used in other readme/documentation. So I added this note. Maybe it's a good idea to rename the folder?

File: Common/Readme.md (hunk 1 above)
Comment (ThirtySomething) [reply Common/Readme.md]: Typo fixed in commit 077f588.

File: Documentation/Readme.md (hunk 2 above)
Comment (DonatJR) [reply Documentation/Readme.md:1]: Can't think of a better name of the top of my head, but if you have something in mind... :)

**Review by DonatJR — APPROVED** [review]
